loader.rb

```diff
@@ -9,15 +9,55 @@
 
 require "bundler/setup"
 Bundler.setup
-require "zeitwerk"
-Loader = Zeitwerk::Loader.new
-Loader.push_dir("#{__dir__}/")
-Loader.push_dir("#{__dir__}/model")
-Loader.push_dir("#{__dir__}/lib")
-Loader.ignore("#{__dir__}/routes")
-Loader.ignore("#{__dir__}/migrate")
-Loader.ignore("#{__dir__}/spec")
-Loader.ignore("#{__dir__}/model.rb")
-Loader.inflector.inflect("db" => "DB")
-Loader.enable_reloading
-Loader.setup
+
+require_relative "./lib/casting_config_helpers"
+require_relative "./config"
+
+require "rack/unreloader"
+
+Unreloader = Rack::Unreloader.new(
+  reload: Config.development?,
+  autoload: !Config.production?
+) { Clover }
```

Comment:
I think `logger` is useful at development. It show which files loaded/unloaded.

```suggestion
require "logger"

Unreloader = Rack::Unreloader.new(
  logger: Config.development? ? Logger.new($stdout) : nil,
  reload: Config.development?,
  autoload: !Config.production?
) { Clover }
```


```suggestion
if Config.development?
  require "logger"
  logger = Logger.new($stdout)
end

Unreloader = Rack::Unreloader.new(
  logger: logger,
  reload: Config.development?,
  autoload: !Config.production?
) { Clover }
```

Reply:
It's kinda verbose though, isn't it? I had it and took it out...it's sort of a lot for `./bin/pry` executions. Maybe let's leave it in for a while.